ts/models/conversation.ts

```diff
@@ -718,6 +725,127 @@ export class ConversationModel extends Backbone.Model<ConversationAttributes> {
     }
   }
 
+  public async sendReactionJob(sourceMessage: MessageModel, reaction: Reaction) {
+    try {
+      const destination = this.id;
+      // TODO update sendToOpenGroupV2 to not need this
+      const fileIdsToLink: Array<any> = [];
+
+      const sentAt = sourceMessage.get('sent_at');
+      if (!sentAt) {
+        throw new Error('sendReactMessageJob() sent_at must be set.');
+      }
+
+      if (this.isPublic() && !this.isOpenGroupV2()) {
+        throw new Error('Only opengroupv2 are supported now');
+      }
+
+      let sender = UserUtils.getOurPubKeyStrFromCache();
```

Comment:
If you look below this if statement there is an else case which also requires the sender. Also since the sender is overwritten if it is blinded I made it let. Let me know if you are happy with this.